notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>Sam wykres z liczbą podmiotów jest ok. Średnia liczba szkół wydaje mi się że powinna mieć jakiś&nbsp;punkt odniesienia - tutaj może przydałby się&nbsp;osobny wykres z liczbą szkół/liczbę mieszkańców (może wtedy mazowieckie wcale nie będzie tak mocno odstawać).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>